src/components/Figure/index.js

```diff
@@ -15,6 +15,7 @@ export { default as FigureByline } from './Byline'
 
 const styles = {
   figure: css({
+    lineHeight: 0,
```

Comment:
could this be moved to figure image? Because line height are inherited it can be a bit dangerous in element that do have text children

Reply:
`line-height:0` on the image and image wrappers doesn't help, but I've now switched to resetting the image to `display: block` instead of using line-height. So both the "native" and the placeholder images are `display: block` now.